crane/container.go

```diff
@@ -122,7 +122,9 @@ func (c *container) Dependencies() *Dependencies {
 	dependencies := &Dependencies{}
 	for _, link := range c.RunParams.Link() {
 		linkParts = strings.Split(link, ":")
-		dependencies.All = append(dependencies.All, linkParts[0])
+		if !dependencies.includesAsKind(linkParts[0], "link") {
```

Comment:
I don't think we understand each other. My assumptions are:
1st) `--link` can be used to create multiple aliases. Mathematically its effect is similar to addition.

2nd) There is no point to support multiple `--volumes-from` of same container. It would result in mapping volumes over mapped volumes of that same container. Mathematically it resembles multiplication.

3rd) Since `--link` is an addition it must be handled specially. Using same pattern as for `--volumes-from` would result in addition of only the first `--link` occurence for each container.
That would prevent the error, but is undesired since it would also disable the feature I am trying to use.

Please correct me if I am wrong!